tests/js/spec/utils/withProjects.spec.jsx

```diff
@@ -9,23 +8,28 @@ describe('withProjects HoC', function () {
     act(() => ProjectsStore.reset());
   });
 
-  it('works', function () {
-    const MyComponent = () => null;
-    const Container = withProjects(MyComponent);
-    const wrapper = mountWithTheme(<Container />);
+  function Output({projects, loadingProjects}) {
+    if (loadingProjects) {
+      return <p>Loading</p>;
+    }
+    return (
+      <p>
+        {projects.map(project => (
+          <span key={project.slug}>{project.slug}</span>
+        ))}
+      </p>
+    );
+  }
 
-    expect(wrapper.find('MyComponent').prop('projects')).toEqual([]);
-    expect(wrapper.find('MyComponent').prop('loadingProjects')).toEqual(true);
+  it('works', async function () {
+    const Container = withProjects(Output);
+    render(<Container />);
+    expect(await screen.findByText('Loading')).toBeInTheDocument();
 
     // Insert into projects store
     const project = TestStubs.Project();
     act(() => ProjectsStore.loadInitialData([project]));
```

Comment:
do we need `act` here?

Reply:
The test fails without it due to an act() warning as the store updates components via listeners.